packages/next-auth/src/css/index.css

```diff
@@ -112,20 +114,22 @@ a.button {
   margin: 0 0 0.75rem 0;
   padding: 0.75rem 1rem;
   color: var(--provider-color, var(--color-primary));
-  background-color: var(--provider-bg, var(--color-background));
+  background-color: var(--provider-bg, var(--color-background-card));
   font-size: 1.1rem;
   min-height: 62px;
   border-color: rgba(0, 0, 0, 0.1);
   border-radius: var(--border-radius);
   transition: all 0.1s ease-in-out;
-  box-shadow: #000 0px 0px 0px 0px, #000 0px 0px 0px 0px,
-    rgba(0, 0, 0, 0.2) 0px 10px 15px -3px, rgba(0, 0, 0, 0.1) 0px 4px 6px -4px;
   font-weight: 500;
   position: relative;
   display: flex;
   align-items: center;
   justify-content: center;
 
+  @media (max-width: 450px) {
+    font-size: 0.9rem;
+  }
+
   &:has(img) {
```

Comment:
`:has()` is not supported on Firefox, making the button image look tight. Can we make improvements for Firefox here?
<img width="535" alt="Screen Shot 2022-11-30 at 22 04 11" src="https://user-images.githubusercontent.com/31528554/204833246-0f0f8cb2-7efd-46d3-8480-03b1622ffc5f.png">

Reply:
Fixed now.
![image](https://user-images.githubusercontent.com/46612252/204847064-09f8d3d4-55b7-4c14-9e5e-6514ac07b6ee.png)

Targeting the span within the button works fine without the has conditional.